docs/source/calico-neutron-api.rst

```diff
@@ -144,3 +153,69 @@ to create one will fail.
 
 .. note:: It is possible that in a future version of Calico LBaaS may be
           functional. Watch this space.
+
+
+.. _horizon-actions:
+
+Horizon
+-------
+
+Horizon makes many provisioning actions available that mirror options on the
+Neutron API. This section quickly lists them, and indicates whether they can
+be used or not, and any subtleties that might be present in them.
+
+Much of the detail has been left out of this section, and is instead present in
+the relevant Neutron API sections above: please consult them for more.
+
+Section: Project
+~~~~~~~~~~~~~~~~
+
+Tab: Compute -> Instances
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When launching instances, remember that security groups are used to determine
+reachability, not networks. Choose networks based on whether you need an
+external or an internal IP address, and choose security groups based on the
+machines you'd like to talk to in the cloud. See
+:ref:`opens-external-conn-setup` for more.
+
+Tab: Compute -> Access & Security
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As noted above, tenants should ensure they configure their security groups to
+set up their connectivity appropriately.
+
+Tab: Network -> Network Topology
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For the 'Create Network' button, see the :ref:`neutron-api-networks` section.
+For the 'Create Router' button, see the :ref:`neutron-api-routers` section.
+
+Tab: Network -> Networks
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+For networks and subnets, see the sections :ref:`neutron-api-networks` and
+:ref:`neutron-api-subnets`.
+
+Tab: Network -> Routers
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Tenants should not be able to create routers, as they serve no purpose in a
```

Comment:
What do you mean by "should not be able to"?  Is this something that Horizon can be set to police (through quotas) or did you really mean "should not as the attempt will fail and it is pointless"?